packages/docs/src/components/Navbar/MobileSidebar/styles.module.css

```diff
@@ -0,0 +1,52 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+
+  top: 80px;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  padding: 24px;
+
+  transform: translateY(-130%);
+  /* transition: transform 0.3s cubic-bezier(0.4, 0, 0.2, 1); */
+
+  background: var(--swm-background);
+  background-color: transparent;
+  backdrop-filter: blur(20px);
+
+  z-index: 999;
+}
+
+.open {
+  transform: translateX(0);
+}
+
+.mobileNavLinks {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.mobileNavLinks a {
+  color: var(--ifm-color-primary);
+  text-decoration: none;
+  font-size: 32px;
+  font-weight: 400;
+  transition: color 0.2s;
+}
+
+.mobileNavLinks a:hover {
+  color: #0070f3;
```

Comment:
hard coded color and the color isn't even correct (should've been green)
<img width="393" height="418" alt="image" src="https://github.com/user-attachments/assets/414b410f-d046-4394-8da6-7ab633e09e2a" />